db/scripts/cleanup_scheduled_calls.js

```diff
@@ -0,0 +1,27 @@
+async function getDb() {
+  const dotenv = require("dotenv");
```

Comment:
If you did use the AppContainer you'd still need to import the dotenv library.

Newer JS syntax for this would be 
```javascript 
import dotenv from 'dotenv';

dotenv.config();
```

Reply:
This results in `SyntaxError: Cannot use import statement outside a module`. Most likely because the script is being run as is and not transpiled